<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#DCDCDC	</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">background-color: #2f3e40; /*#2f3e46,1a535c;1d3557;233d4d*/
color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Ajout Monnaie</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>466</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Text">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ButtonText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="PlaceholderText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Text">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ButtonText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="PlaceholderText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Text">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ButtonText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>220</red>
                     <green>220</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="PlaceholderText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>47</red>
                     <green>62</green>
                     <blue>70</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #2f3e46;</string>
               </property>
               <property name="text">
                <string>Code Monnaie</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #2f3e46;</string>
               </property>
               <property name="text">
                <string>Libellé Monnaie</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true">color: #2f3e46;</string>
               </property>
               <property name="text">
                <string>Taux de Change</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="code_monnaie">
               <property name="styleSheet">
                <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="libelle_monnaie">
               <property name="styleSheet">
                <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="taux_change">
               <property name="styleSheet">
                <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="ajouter_ajout_monnaie">
             <property name="styleSheet">
              <string notr="true">QPushButton {width: 65px;
background-color: white;
border: 2px solid #3bb273;
color: #3bb273;
padding: 6px;
border-radius: 10px;}

QPushButton:hover {background-color: #3bb273;color: white;}
QPushButton:pressed { margin: 1px; }
</string>
             </property>
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fermer_ajout_monnaie">
             <property name="styleSheet">
              <string notr="true">QPushButton {background-color: white;
border: 2px solid #ee2e31;
color: #ee2e31;
padding: 5px;
border-radius: 10px;}
QPushButton:hover {
background-color: #ee2e31;
color: white;
}
QPushButton:pressed{
margin: 1px;
}
/*#ee2e31*/</string>
             </property>
             <property name="text">
              <string>Fermer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
